src/array-methods/1/task.js

```diff
@@ -0,0 +1,11 @@
+const flowers = [
+    { name: "Rose", color: "red", price: 15, availability: 20 },
+    { name: "Tulip", color: "yellow", price: 8, availability: 5 },
+    { name: "Orchid", color: "purple", price: 25, availability: 10 }
+];
+
+let expensiveFlowers = flowers.filter(flower => flower.price > 10)
```

Comment:
1. Doesn't seem that `let` is required here - or am I missing something? If not, please change to `const` here, and in any other places in the exercises.

2. The implementation is good, however, whenever possible, you can skip creating the variables, and just chain the funtions, so it would be like
```
const flowerNamesUpperCase = flowers.filter(...)
    .sort(...)
    .map(...)
```

That way you're saving JS from having to create some vars, and also make the code clearer. It's not a mistake to do it as you did tho. We however do it only if there's a need to re-use that data somewhere - then we assign it to a variable.